global.json

```diff
@@ -1,12 +1,12 @@
 {
   "tools": {
-    "dotnet": "8.0.100-preview.6.23330.14",
+    "dotnet": "7.0.306",
     "runtimes": {
       "dotnet": [
-        "7.0.1"
+        "7.0.9"
```

Comment:
NIT: You no longer need the runtimes section. It was only required before because we were using a .NET 8 SDK which only comes with .NET 8 sharedfx, so we needed to additionally restored 7.0. Now that we are using 7.0 SDK, sharedFX should be included so no need for that section.

Reply:
Should I also set `sdk: ...` ?

Reply:
You can if you want. The tools section is mostly just so that our tooling will restore it automatically, the sdk property is so that people that don't have this version installed won't be able to call dotnet commands in the repo (essentially you force people to have that sdk or higher.)